### ISSUE TITLE
How about looking up contact info at thatsthem.com

### ISSUE THREAD

**Issue by stocks81** [kawaiipantsu/maltego-darknet-transforms#3]
They have a lookup for people, phone, address, email, and IP address.   Might be a great tool to use to try and find people.  I have used the site for some time, the info is some times out of date, but might be a great point to help lookup info. [https://thatsthem.com/](https://thatsthem.com/)

**Comment by kawaiipantsu** [comment]
I totally agree, "the more the merrier" to collect usable data is better. I'm also looking for DK (Danish) sources as i'm from Denmark. So i might create transform-sugestions with that as well, if you know any just create issues on em.

Also - Really i'm looking for transforms to play with to get the feel on how it works. So i'm very much interested in "other" transforms to lookup, other than just the once i have on the drawing board. Again, i know "Darknet" might be misleading but it's my domain name heh darknet.dk so hence the darknet in the name .. But i'm also aiming for like Tor exit node lookup, Tor address lookup (via proxy), Digging for web site platform and backend server and so on. I'm creating issues along for all of them.
